clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/model/ManifestProvider.java

```diff
@@ -27,6 +27,17 @@
 
   T getManifest(String account, String location, String name);
 
+  T getManifest(String account, String location, String name, boolean includeEvents);
+
   List<T> getClusterAndSortAscending(
       String account, String location, String kind, String app, String cluster, Sort sort);
+
+  List<T> getClusterAndSortAscending(
```

Comment:
This is only ever called in one place (`getDynamicManifestFromCluster`); the only existing call to that REST endpoint is from `DynamicResolveManifestTask` in `orca` which also doesn't use the events, so I think we can safely just always omit events from this call.  If someone adds a use case that needs the events, they can add in the flag and pass it through.  (Notably this is never exposed through gate, only internal REST endpoints which we don't guarantee backwards compatibility for.)